help/md/dnExponential.md

```diff
@@ -3,13 +3,9 @@ dnExponential
 ## title
 Exponential Distribution
 ## description
-Exponential distribution with rate equal to ‘lambda’.
+The Exponential probability distribution describes the distribution of the times between events in a Poisson point process. 
```

Comment:
Instead of `Exponential _probability_ distribution`, lets just write `Exponential distribution`.  Also in other help files.